nodejs/week2/mealsharing/src/backend/routes/meals.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require('../data/meals.json');
+let meal;
+router.get('/:id', (req, res) => {
+  meal = meals.find((items) => items.id === parseInt(req.params.id));
+  res.send(meal);
+});
+
+router.get ('/', (req, res)=>{
```

Comment:
I think, you need to define meal in the second route. let meal;
Then assign value to meal based on condition.